content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -30,6 +30,12 @@ spec:
         ISTIO_META_DNS_CAPTURE: "true"
         # Enable automatic address allocation, optional
         ISTIO_META_DNS_AUTO_ALLOCATE: "true"
+    # The below configuration is only used in the External TCP services section of this document
+    # to clarify the external service behaviour in a simplified way.
+    # Otherwise this is not a mandatory setting for DNS proxying.
+    discoverySelectors:
+    - matchLabels:
+        istio-injection: enabled
```

Comment:
I think it would be better to leave this out here, and just keep it as simple as possible for using the feature. You will then need to repeat the install command in the later section, but that's usually preferable anyway vs telling the user to go back and follow instructions in an earlier section.

```suggestion
```